.travis.yml

```diff
@@ -17,8 +17,19 @@ before_install:
 install:
   - ./setup.py install
 
+before_script:
+  - mkdir exec
+  - cd exec
+  - git clone https://github.com/fossology/fossology
+  - cd fossology/src/nomos/agent/
+  - make -f Makefile.sa
+  - cd /home/travis/build/chaoss/grimoirelab-graal/exec
+  - wget https://github.com/nexB/scancode-toolkit/releases/download/v3.0.0/scancode-toolkit-3.0.0.zip
+  - unzip -q scancode-toolkit-3.0.0.zip
```

Comment:
I have run some local tests and it seems that scancode needs to be execute the first time to be configured. Thus after the unzip command, the script should enter in `scancode-toolkit-3.0.0` directory and execute `./scancode --help`.

Reply:
@valeriocos Sorry i missed that out 🙈. I've updated the file.